crypto/aes/build.info

```diff
@@ -38,6 +38,8 @@ GENERATE[aesv8-armx.S]=asm/aesv8-armx.pl $(PERLASM_SCHEME)
 INCLUDE[aesv8-armx.o]=..
 GENERATE[vpaes-armv8.S]=asm/vpaes-armv8.pl $(PERLASM_SCHEME)
 
+GENERATE[aes-armv4.S]=asm/aes-armv4.pl $(PERLASM_SCHEME)
+GENERATE[bsaes-armv7.S]=asm/bsaes-armv7.pl $(PERLASM_SCHEME)
```

Comment:
I take back that comment...  but, have you reconfigured (possibly automatically)?  Otherwise, you may end up with a Makefile that's not up to date.  `./Configure reconf`, just to be safe